scheduler/cmd/pipelinegateway/main.go

```diff
@@ -86,16 +97,7 @@ func init() {
 	flag.IntVar(&schedulerTlsPort, flagSchedulerTlsPort, defaultSchedulerTLSPort, "Scheduler TLS port")
 	flag.StringVar(&envoyHost, flagEnvoyHost, "0.0.0.0", "Envoy host")
 	flag.IntVar(&envoyPort, flagEnvoyPort, defaultEnvoyPort, "Envoy port")
-}
-
-func makeSignalHandler(logger *log.Logger, done chan<- bool) {
-	exit := make(chan os.Signal, 1)
-	signal.Notify(exit, os.Interrupt, syscall.SIGTERM)
-
-	<-exit
-
-	logger.Info("shutting down due to SIGTERM or SIGINT")
-	close(done)
+	flag.IntVar(&healthProbeServicePort, flagHealthPort, defaultHealthProbePort, "K8s health probe port")
```

Comment:
```suggestion
	flag.IntVar(&healthProbeServicePort, flagHealthPort, defaultHealthProbePort, "Health probe port")
```
Suggestion because Core 2 will happily run in something like docker compose if needed, it's just that we don't really support those deployment types for production atm.